packages/grafana-data/src/themes/createTypography.ts

```diff
@@ -115,8 +115,8 @@ export function createTypography(colors: ThemeColors, typographyInput: ThemeTypo
     h4: buildVariant(fontWeightRegular, 18, 1.235, 0.25),
     h5: buildVariant(fontWeightRegular, 16, 1.334, 0),
     h6: buildVariant(fontWeightMedium, 14, 1.6, 0.15),
-    body: buildVariant(fontWeightRegular, 14, 1.5, 0.15),
-    bodySmall: buildVariant(fontWeightRegular, 12, 1.5, 0.15),
+    body: buildVariant(fontWeightRegular, 14, 1.57142857143, 0.15),
```

Comment:
think we should:
- add a comment explaining that these `font-size`/`line-height` values need to be an even number else it causes alignment issues
- adjust the other font-sizes/line-height ratios to be even numbers, e.g.
```typescript
  const variants = {
    h1: buildVariant(fontWeightLight, 28, 1.14285714286, -0.25),
    h2: buildVariant(fontWeightLight, 24, 1.166666, 0),
    h3: buildVariant(fontWeightRegular, 22, 1.181818, 0),
    h4: buildVariant(fontWeightRegular, 18, 1.222222, 0.25),
    h5: buildVariant(fontWeightRegular, 16, 1.375, 0),
    h6: buildVariant(fontWeightMedium, 14, 1.57142857143, 0.15),
    body: buildVariant(fontWeightRegular, 14, 1.57142857143, 0.15),
    bodySmall: buildVariant(fontWeightRegular, 12, 1.5, 0.15),
  };
```

can't notice anything obviously wrong with these values and it will hopefully make us more consistent going forward

Reply:
Nice one @torkelo and nice catch @ashharrison90 ! Agree with ash here about adjusting the other ones and also leaving a comment so we're not confused about it looking like a random number 😄